nixos/modules/services/networking/uptermd.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.uptermd;
+in
+{
+  options = {
+    services.uptermd = {
+      enable = mkEnableOption "uptermd";
+
+      openPorts = mkOption {
```

Comment:
Every other module calls this `openFirewall`.

Reply:
for consistency I also recommend using ```openFirewall``` instead of ```openPorts```

Reply:
done